docs/src/pages/playground.tsx

```diff
@@ -26,23 +35,30 @@ function Homepage() {
         headers: {
           'Content-Type': 'application/json',
         },
-        body: textAreaValue,
+        body: `{ "scene": ${textAreaValue} }`,
```

Comment:
```suggestion
        body: JSON.stringify({
          scene: JSON.parse(textAreaValue)
        }),
```

Separate:
- code that makes fetch request and builds error
- logic that builds full request object (adds that scene)
- handling error/success of the request (setting the state)